client/src/snippets.json

```diff
@@ -2,10 +2,21 @@
     "Basic Galaxy tool": {
         "prefix": "gx-tool",
         "body": [
-            "<tool id=\"$1\" name=\"$2\" version=\"0.1.0\">",
+            "<tool id=\"$1\" name=\"$2\" version=\"0.1.0\" profile=\"20.01\" license=\"TODO_license_in_spdx_format\">",
+            "    <description></description>",
+            "    <xrefs>",
+            "        <xref type=\"bio.tools\"></xref>",
+            "    </xrefs>",
             "    <requirements>",
             "        $0",
             "    </requirements>",
+            "    <edam_topics>",
+            "        <edam_topic></edam_topic>",
+            "        <edam_topic></edam_topic>",
+            "    </edam_topics>",
```

Comment:
Should we keep only one `<edam_topic></edam_topic>` here and maybe add a comment like:
````xml
<edam_topics>
    <!-- TODO: please annotate this tool with topics from http://edamontology.org -->
    <edam_topic></edam_topic>
</edam_topics>
````

Reply:
Yes, this sounds like a good idea!

Reply:
Added,